orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/IgorService.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2015 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.orca.igor;
+
+import com.netflix.spinnaker.kork.artifacts.model.Artifact;
+import retrofit.http.*;
+
+import java.util.List;
+import java.util.Map;
+
+public interface IgorService {
+  @PUT("/masters/{name}/jobs/{jobName}")
+  public abstract String build(@Path("name") String master, @Path(encode = false, value = "jobName") String jobName, @QueryMap Map<String, String> queryParams, @Body String ignored);
```

Comment:
Line length for all these? Please split each arg onto its own line :)

Reply:
Good point, will fix.  Also just realized that the `public abstract` part is redundant on an interface so will remove.